content/version/1/4/code-of-conduct.is.md

```diff
@@ -0,0 +1,77 @@
++++
+version = "1.4"
+aliases = ["/version/1/4/is"]
++++
+
+# Hátternisreglur þátttakenda
+
+## Loforð okkar
+
+Í þágu þess að hlúa að opnu og aðlaðandi umhverfi, viljum við sem þátttakendur
+og umsjónarfólk heita því að gera þátttöku í verkefni okkar og félagsskap
+upplifun án áreitis fyrir alla, óháð aldri, líkamsstærð, fötlunar, þjóðerni,
+kynvitundar og tjáningar, reynslu, ríkisfangs, útlits, kynþáttar, trúar eða
+kyneinkenna og kynhneigðar.
+
+## Kröfur okkar
+
+Dæmi um hegðun sem stuðlar að jákvæðu umhverfi:
+
+* Notkun á opnu og aðlaðandi tungumáli
+* Virðing fyrir mismunandi sjónarhornum og reynslu
+* Taka vel á móti uppbyggjandi gagnrýni
+* Leggja áherslu á það sem er best fyrir samfélagið
+* Sýna öðrum meðlimum samkennd
+
+Dæmi um óviðunandi hegðun þáttakenda:
+
+* Notkun kynferðislegs máls eða myndmáls og óvelkomin kynferðisleg athygli eða
+  áreitni
+* „Trolling“, móðgandi/niðrandi ummæli og persónulegar eða pólitískar árásir
+* Áreitni í hvaða formi sem er
+* Útgáfa á persónulegum upplýsingum annara, svo sem eiginlegs eða rafræns
+  heimilisfangs án skýrs leyfis
+* Önnur hegðun sem gæti innan skynsamlegra marka verið talin óviðeigandi í
+  faglegu samhengi
+
+## Ábyrgð okkar
+
+Umsjónarfólk verkefnisins er ábyrgt fyrir því að setja fram skýrar kröfur um
+ásættanlegra hegðun og gert er ráð fyrir að það gerðar séu viðeigandi og
+sanngjarnar úrbætur í svari við öllum tilvikum óviðunandi hegðunnar.
+
+Umsjónarfólk verkefnis hefur rétt og ber skylda til að fjarlægja, breyta, eða
+hafna athugasemdum, „commitum“, kóða, wiki breytingum, málum, og öðrum framlögum
+sem ekki eru í takt við þessar hátternisreglur, eða að banna tímabundið eða
+varanlega hvern þann þáttakenda fyrir hverja þá hegðun sem það telur
+óviðeigandi, ógnandi, dónalega, eða skaðlega.
+
+## Umfang
+
+Þessar hátternisreglur eiga bæði við innan verkefnis og í opinberum rýmum þar
+sem einstaklingur er fulltrúi verkefnis eða samfélags þess. Dæmi um það að vera
+fulltrúi verkefnis eða samfélags tekur til notktunar á opinberum netföngum
+verkefnis, skrif á samfélagsmiðlum, eða starfa sem skipaður fulltrúi á atburði,
+hvort sem það er á netinu eða í raunheimum. Það að vera fulltrúi verkefnis getur
+verið frekar skilgreint og skýrt af umsjónarfólki verkefnis.
+
+## Fullnusta
+
+Tilvik af ofbeldisfullri, áreitni, eða annarskonar óviðunandi hegðunnar geta
+verið tilkynntar með því að hafa samband við verkefnateymi í [NETFANG HÉR].
+Allar kvartanir verða yfirfarnar og rannskaðar og munu leiða til viðbragða sem
+eru talin nauðsynleg og viðeigandi fyrir aðstæður. Verkefnateymi er skylt til að
+halda trúnað er bundið trúnaði við þann sem tilkynnir atvik. Frekari upplýsingar
```

Comment:
`skylt til að halda trúnað er bundið trúnaði` seems redundant/wrong

Reply:
Yeah, that's me not cleaning up 😆